### PR TITLE
only ".gitattributes" + whitespace

### PR DESCRIPTION
**Pull request by voku**

### CODE REVIEW

**Comment by paragonie-scott** [comment]
Thanks :)

File: lib/byte_safe_strings.php [hunk 1]
```diff
@@ -50,8 +50,10 @@ function RandomCompat_strlen($binary_string)
                     'RandomCompat_strlen() expects a string'
                 );
             }
+
             return mb_strlen($binary_string, '8bit');
         }
+
```
Comment (GrahamCampbell) [inline lib/byte_safe_strings.php]: :-1: for that addition

Comment (glensc) [inline lib/byte_safe_strings.php]: why not setup php-cs-fixer rules that can handle most formatting (trailing eol, newline before return can be controlled by it), and contributors can run it before submiting PR

Comment (GrahamCampbell) [inline lib/byte_safe_strings.php]: Even better, setup https://styleci.io and forget about running local tools.

Comment (voku) [inline lib/byte_safe_strings.php]: :+1: for "styleci.io"

Comment (narfbg) [inline lib/byte_safe_strings.php]: Best option: Contributors don't alter code style to match their own preferences.

Comment (GrahamCampbell) [inline lib/byte_safe_strings.php]: > Best option: Contributors don't alter code style to match their own preferences.

StyleCI bipasses this if autosending and merging is enabled. It'll revert any bad cs changes immidately after PR merging.

Comment (narfbg) [inline lib/byte_safe_strings.php]: I know that, but it's not the point.

This very PR is obviously intended to make such changes, which shouldn't happen. Everybody should just stop assuming that *their* style is what everybody wants.

**Pull request merged**